tests/lti/views/oauth_test.py

```diff
@@ -23,23 +23,24 @@
 
 
 @pytest.mark.usefixtures('util')
-class TestTokenCallback(object):
+@pytest.mark.parametrize('method', [oauth.token_callback, oauth.refresh_callback])
+class TestTokenCallbackAndRefreshCallback(object):
```

Comment:
The `@pytest.mark.parametrize` here means that pytest will call every test method in this class twice - once for the `oauth.token_callback()` method and once for `oauth.refresh_callback()`. `token_callback()` and `refresh_callback()` are passed in to the test methods as the `method` argument.